### DISCUSSION TITLE
distinguishing templates

### DISCUSSION THREAD

**Discussion by kofifus** [Dotnet-Boxed/Templates#201] [Q&A]
Hi newbe question here, how do I distinguish between the templates ? I want the GraphQL one but they seem to both be created with 'dotnet new api' ?

Thanks!

**Comment by RehanSaeed** [comment]
Run:

`dotnet new api --name 'Foo'`

Or:

`dotnet new graphql --name 'Foo'`

**Comment by kofifus** [comment]
thanks ! would be good to update the docs
